cmake/Codegen.cmake

```diff
@@ -1,89 +1,95 @@
-if(Codegen_GPU_cmake_included)
+if(Codegen_XPU_cmake_included)
```

Comment:
That's quite a change. I worry we might see more issues with torch code being updated and 2 things getting out of sync:
* torch-xpu-ops Codegen.cmake and related scripts with torch side versions of the same
* ops code in native/xpu folder which you've modified for some include files

Any chance we can start bringing pieces of this code into torch codebase itself? For example, any chance we can stop having Codegen.cmake here on torch-xpu-ops side?

Note: don't consider above as a request to do that in this PR. I am just trying to discuss.

Reply:
Thanks, @dvrogozh, for your verification. I agree that these two components may be out of sync. However, upstreaming this code to stock PyTorch could be challenging since torch-xpu-ops is out of tree. We can track any out-of-sync issues through our nightly builds and CI unit tests. For now, we may need to address and maintain them ourselves.